app/views/spree/admin/orders/_add_product.html.haml

```diff
@@ -4,8 +4,11 @@
   %fieldset.no-border-bottom
     %legend{:align => "center"}= Spree.t(:add_product)
 
-    .field.twelve.columns.alpha{"data-hook" => "add_product_name"}
-      = label_tag :add_variant_id, Spree.t(:name_or_sku)
-      = hidden_field_tag :add_variant_id, "", :class => "variant_autocomplete fullwidth"
-      
+    - if @order.canceled?
+      = Spree.t(:cannot_add_item_to_canceled_order)
```

Comment:
I think that we should use away from the `Spree.t` method and use lazy lookup instead.
```suggestion
      = t(".cannot_add_item_to_canceled_order")
```
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-%28i18n%29#development

Reply:
I'm wondering here though if we should just take out the whole section. We think that nobody needs to change a cancelled order and therefore we don't need to tell people that they can't. Nobody would think of doing it.

Providing a screenshot of your change is always helpful.